kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/InputFormatUtil.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2013 Cloudera Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kitesdk.data.spi.filesystem;
+
+import org.apache.hadoop.mapreduce.RecordReader;
+import org.apache.hadoop.mapreduce.lib.input.FileInputFormat;
+import org.kitesdk.compat.DynConstructors;
+import org.kitesdk.data.DatasetDescriptor;
+import org.kitesdk.data.DatasetException;
+
+public class InputFormatUtil {
+  public static final String INPUT_FORMAT_CLASS_PROP = "kite.inputformat.class";
```

Comment:
Would there be any value in having the format also support a SerDes class? Essentially this format is half of what the read path in Hive offers. That is, you can provide an InputFormat which determines how files are split, but we don't allow anything that changes how records are deserialized other than a transform function during import.

If we did add a SerDes interface, I'd want the target to build Avro records rather than Hive records.

Either way, I don't think we need this for this use case, but it might be worth adding at some point in the future.